python/felis/cli.py

```diff
@@ -417,6 +251,40 @@ def load_tap_schema(
     ).load()
 
 
+@cli.command("init-tap-schema", help="Initialize a standard TAP_SCHEMA database")
+@click.option("--engine-url", envvar="FELIS_ENGINE_URL", help="SQLAlchemy Engine URL")
+@click.option("--tap-schema-name", help="Name of the TAP_SCHEMA schema in the database")
```

Comment:
Is it perhaps worth documenting that this defaults to TAP_SCHEMA if left empty?

Reply:
Good idea. I will add a note on this.